lib/lita/services/lunch_assigner.rb

```diff
@@ -100,6 +100,16 @@ def persist_winning_lunchers
         end
       end
 
+      def persist_wagers
+        sw = Lita::Services::SpreadsheetManager.new(ENV.fetch('GOOGLE_SP_DB_KEY'),'wagers')
+        time = Time.now.strftime('%Y-%m-%d')
+        current_lunchers_list.each do |luncher|
+          user = User.find_by_mention_name(luncher) || User.find_by_name(luncher)
```

Comment:
Use find_by instead of dynamic find_by_mention_name.<br>Use find_by instead of dynamic find_by_name.